Project B/CinemaHall1.cs

```diff
@@ -1,9 +1,10 @@
-class CinemaHall1 : CinemaHall 
+class CinemaHall1 : CinemaHall
 {
     public CinemaHall1(int id) : base(id, 150)
     {
-        FillChairs();
+        FillChairs(id, "name", 20.0);
```

Comment:
Nu wordt elke stoel 20 euro, maar dat hoort toch niet?

Reply:
is een placeholder totdat we die prijzen hebben ingesteld enzo, dan kan ik die functie uitbreiden om alle factoren te includen